bot/data/CreateTables.sql

```diff
@@ -115,3 +115,10 @@ CREATE TABLE IF NOT EXISTS ClaimsMapping (
         REFERENCES Guilds (id)
 );
 
+CREATE TABLE IF NOT EXISTS Reminders (
+    id                      VARCHAR(255)     NOT NULL,
+    user                    INTEGER          NOT NULL,
+    message                 TEXT             NOT NULL,
```

Comment:
we need ID to be a primary key and the user and message fields to foreign keys. if you have any questions ping me